setup.py

```diff
@@ -89,8 +91,10 @@ def parse_arg_remove_string(argv, arg_name_equal):
 elif parse_arg_remove_boolean(sys.argv, "--use_qnn"):
     package_name = "onnxruntime-qnn"
 
-if is_rocm or is_migraphx:
+if is_rocm:
     package_name = "onnxruntime-rocm" if not nightly_build else "ort-rocm-nightly"
+elif is_migraphx:
+    package_name = "onnxruntime-migraphx" if not nightly_build else "ort-migraphx-nightly"
```

Comment:
We should stick with the same naming here as onnxruntime-rocm

Reply:
With this PR, if MIGraphX EP and ROCm EP are built in the same build, the package will have the `onnxruntime-rocm` name. However, when building only MIGraphX EP and using the same package name as for ROCm EP, it will collide when installing (pip will refuse to install) - one can build independently ROCm EP and then as a separate build MIGraphX EP and then install both (that is the official way of how different providers should be installed). Forcing reinstall will remove ROCm. We cannot forbid building MIGraphX EP without ROCm EP because projects (i.e., UAI) might need only MIGraphX EP.

Reply:
Sounds like a lot of the issues we're having now is because the original use case was to bundle both EPs together similar to Nvidia (CUDA + Tensor RT EPs)